src/stories/inputbox.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import InputBox from "~/components/inputbox"; // Update the path to where your InputBox component is located
+
+const meta: Meta<typeof InputBox> = {
+  title: "InputBox",
+  component: InputBox,
+  argTypes: {
+    variant: {
+      options: ["primary", "secondary"],
+      control: {
+        type: "select",
+      },
+    },
+    placeholder: {
+      control: "text",
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof InputBox>;
+
+export const Name: Story = {
```

Comment:
For storybook, you don't need separate components with different placeholder values (that can be done from inside storybook itself)